dipy/workflows/io.py

```diff
@@ -475,8 +475,7 @@ def run(
                             "Nifti or Trk file using the option "
                             "--reference my_files.nii.gz ."
                         )
-                        logging.error(msg, stacklevel=2)
-                        sys.exit(1)
+                        raise IOError(msg)
```

Comment:
Can you keep it as it was before. 

We realize that non-developer panic when we raise an exception. too much information. 

That's why we just log a message and exit in a clean way for workflows and keep raising exception for the library (more for the developer)

Maybe, there is a way to avoid the logtrace when you raise the exception.